src/config/database.ts

```diff
@@ -1,22 +1,38 @@
-import { DataSource } from 'typeorm';
+import mongoose from 'mongoose';
 
-export const MongoDataSource = new DataSource({
-  name: 'default',
-  type: 'mongodb',
-  authSource: 'admin',
-  database: process.env.DB_DATABASE,
-  url: process.env.DB_CONNECTION_STRING,
-  entities: ['src/entity/*.ts', 'entity/*.js'],
-  logging: true,
-  synchronize: true
-});
+export const connectDB = async () => {
+  try {
+    await mongoose.connect(process.env.DB_CONNECTION_STRING, {
+      dbName: process.env.DB_DATABASE // Especifica o nome do banco de dados
+    });
+    console.log('MongoDB connected successfully!');
+  } catch (error) {
+    console.error('MongoDB connection error:', error);
+    process.exit(1);
+  }
+};
 
-export const MysqlDataSource = new DataSource({
-  name: 'default',
-  type: 'mysql',
-  database: process.env.DB_DATABASE,
-  url: process.env.DB_CONNECTION_STRING,
-  entities: ['src/entity/*.ts', 'entity/*.js'],
-  logging: true,
-  synchronize: true
-});
+// import { DataSource } from 'typeorm';
```

Comment:
Não deixar o código comentado, se ele não for ser mais utilizado, remova-o.
O histórico de commits não deixará você perder o código que já foi uma vez utilizado.
Por isso não precisa se preocupar, se for necessário utilizar o código novamente, só acessar ele pelo histórico de alterações/commits